packages/flutter/lib/src/widgets/pop_scope.dart

```diff
@@ -49,7 +49,7 @@ import 'routes.dart';
 ///    back gestures in the case of a form with unsaved data.
 ///  * [ModalRoute.registerPopEntry] and [ModalRoute.unregisterPopEntry],
 ///    which this widget uses to integrate with Flutter's navigation system.
-class PopScope extends StatefulWidget {
+class PopScope<T> extends StatefulWidget {
```

Comment:
Nit: Can you document what the `T` parameter means? Maybe something like this, based on MaterialPageRoute's docs:

 > The type parameter `T` represents the return type of the route in which this widget appears. This result can be supplied as the route is popped from the stack with [Navigator.pop] by providing the optional `result` argument. It will be passed into [onPopInvoked] as the `result` argument with the specified type.